docs/book/user-guide/production-guide/cloud-orchestration.md

```diff
@@ -41,34 +41,36 @@ In order to launch a pipeline on AWS with the SkyPilot orchestrator, the first
 thing that you need to do is to install the AWS and Skypilot integrations:
 
 ```shell
-zenml integration install aws skypilot_aws -f
+zenml integration install aws skypilot_aws -y
 ```
 
 Before we start registering any components, there is another step that we have 
 to execute. As we [explained in the previous section](./remote-storage.md#configuring-permissions-with-your-first-service-connector), 
 components such as orchestrators and container registries often require you to 
 set up the right permissions. In ZenML, this process is simplified with the 
 use of [Service Connectors](../../stacks-and-components/auth-management). 
-For this example, we will go ahead and use the [implicit authentication feature 
-of our AWS service connector](../../stacks-and-components/auth-management/aws-service-connector.md#implicit-authentication) 
+For this example, we will go ahead and use the [IAM role authentication method 
+of our AWS service connector](../../stacks-and-components/auth-management/aws-service-connector.md#aws-iam-role) 
 if you haven't already created a service connector in the last section:
```

Comment:
I think we need to take out this line and explicitly say that we need to create a new service connector